src/x86_64/region.h

```diff
@@ -41,16 +42,16 @@ static inline u64 allocate_region(heap h, bytes size)
     u64 base = 0;
     region r;
 
-    /* Select the highest physical region that's within 32-bit space. */
```

Comment:
I suppose the original intent here was to try and keep the identity heap out of the way of low address space that the process might be using. As long as this doesn't break our other platforms, it should be ok for now. We really ought to move all kernel mappings to high (> 2^47) address space.